src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/MultisigCreationFees.tsx

```diff
@@ -62,7 +62,7 @@ export const MultisigCreationFees = memo(
     }, []);
 
     if (isLoading) {
-      return <FeeLoader fiatFlag={Boolean(fiatFlag)} />;
+      return <FeeLoader className="mr-4" fiatFlag={Boolean(fiatFlag)} />;
```

Comment:
Maybe just div with `mr-4`? Should be easier, than passing new prop to logical component

Reply:
I'd say it's pretty standard to let any component get styled according to the use-case isn't it? I could obviously wrap in a div, but it seemed to me that it's not what we were doing for components in general. What do you think?

Reply:
I assume the idea is to seal ui components because they're designed absolutely perfect (at least we should try to do so), then use additional wrappers to add space if it's required and not bother about ui component implementation and styles. Seems to me that's an Open-closed principle in UI.
@johnthecat what do you think?

Reply:
got it, I'm unopinionated about this. I've added a wrapper `div`.